docs/_adapters/nginx-sm.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: Nginx Servce Mesh Adapter
+name: Meshery Adapter for Nginx Servce Mesh
+mesh_name: Nginx Servce Mesh
+version: v2.5.0
+port: 10001/tcp
+project_status: stable
+github_link: https://github.com/layer5io/meshery-nginx-sm
+image: /docs/assets/img/service-meshes/nginx-sm.svg
+---
+# {{ page.name }}
+
+| Service Mesh   | Adapter Status | Latest Supported Mesh Version |
+| :------------: | :------------:   | :------------:              |
+| {{page.title}} | [{{ page.project_status }}]({{ page.github_link }}) | {{page.version}}  |
+
+### Lifecycle management
+
+The {{page.name}} can install **{{page.version}}** of {{page.mesh_name}}. A number of sample applications can be installed using the {{page.name}}.
+
+### Features
+1. Lifecycle management of {{page.mesh_name}}
+1. Lifecycle management of sample applications
+1. Performance testing
+
+### Sample Applications
+
+The {{ page.name }} includes a handful of sample applications. Use Meshery to deploy any of these sample applications.
+
+- [Emojivoto](https://github.com/BuoyantIO/emojivoto)
```

Comment:
Do we support this app in the NGINX SM adapter currently or is this future-facing support?